cmd/strapi-demo/createdemodata/start.go

```diff
@@ -0,0 +1,224 @@
+	Email    string `json:"email"`
+	IsAdmin  bool   `json:"isAdmin"`
+}
+
+// dummy data for demo purposes
+var studentRecord1 = map[string]string{ //nolint: gochecknoglobals
+	"studentid":  "1234568",
+	"name":       "Tanu",
+	"university": "Faber College",
+	"semester":   "3",
+	"issuedate":  "2019-01-02T00:00:00.000Z",
+}
+var studentRecord2 = map[string]string{ //nolint: gochecknoglobals
+	"studentid":  "323456898",
+	"name":       "Derek",
+	"university": "Faber College",
+	"semester":   "2",
+	"issuedate":  "2019-03-02T00:00:00.000Z",
+}
+var transcriptRecord1 = map[string]string{ //nolint: gochecknoglobals
+	"studentid":    "323456898",
+	"name":         "Tanu",
+	"university":   "Faber College",
+	"status":       "graduated",
+	"totalcredits": "100",
+	"course":       "Bachelors'in Computing Science",
+}
+var transcriptRecord2 = map[string]string{ //nolint: gochecknoglobals
+	"studentid":    "1234568",
+	"name":         "Derek",
+	"university":   "Faber College",
+	"status":       "graduated",
+	"totalcredits": "200",
+	"course":       "Bachelors'in Computing Science",
+}
+
+// GetStartCmd returns the Cobra start command.
+func GetStartCmd() *cobra.Command {
+	startCmd := createStartCmd()
+
+	createFlags(startCmd)
+
+	return startCmd
+}
+func createStartCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "create-demo-data",
+		Short: "Create demo data",
+		Long:  "Start populating data in strapi with default studentcards and transcripts",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			hostURL, err := cmdutils.GetUserSetVar(cmd, adminURLFlagName, adminURLEnvKey)
+			if err != nil {
+				return err
+			}
+			parameters := &strapiDemoParameters{
+				client:   &http.Client{},
+				adminURL: hostURL,
+			}
+			return startStrapiDemo(parameters)
+		},
+	}
+}
+
+func createFlags(startCmd *cobra.Command) {
+	startCmd.Flags().StringP(adminURLFlagName, adminURLFlagShorthand, "", adminURLFlagUsage)
+}
+
+// For Demo you can verify the records by browsing http://localhost:1337/admin/
+func startStrapiDemo(parameters *strapiDemoParameters) error {
+	if parameters.adminURL == "" {
+		return errMissingAdminURL
+	}
+
+	var client = parameters.client
+
+	adminUserValues := map[string]string{
+		"username": "strapi",
+		"email":    "user@strapi.io",
+		"password": "strapi"}
+
+	authToken, err := createAdminUser(client, parameters.adminURL, adminUserValues)
+	if err != nil {
+		return err
+	}
+
+	createOrFetchRecord(client, authToken, parameters.adminURL+studentCardsEndpoint, "POST", studentRecord1)
+	createOrFetchRecord(client, authToken, parameters.adminURL+studentCardsEndpoint, "POST", studentRecord2)
+	createOrFetchRecord(client, authToken, parameters.adminURL+transcriptEndpoint, "POST", transcriptRecord1)
+	createOrFetchRecord(client, authToken, parameters.adminURL+transcriptEndpoint, "POST", transcriptRecord2)
+	createOrFetchRecord(client, authToken, parameters.adminURL+studentCardsEndpoint+"/1", "GET", studentRecord1)
```

Comment:
createOrFetchRecord function should return response and then you should verify that response against studentRecord1. You should not add verification to createOrFetchRecord. 

This way you don't need global variables so you can get rid of nolint: gochecknoglobals. 

If you want to keep same interface for your function you can do something like:
createOrFetchRecord(client, authToken, parameters.adminURL+studentCardsEndpoint, "POST", studentRecord1)
var cmsSudentRecord1 map[string]interface{}
createOrFetchRecord(client, authToken, parameters.adminURL+studentCardsEndpoint+"/1", "GET", cmsStudentRecord1)
check if cmsStudentRecord1 data matches studentRecord1